src/main/scala/org/pac4j/http4s/SecurityFilterMiddleware.scala

```diff
@@ -51,13 +53,13 @@ object SecurityFilterMiddleware {
         IO[Response[IO]],
         Http4sWebContext
       ] = new DefaultSecurityGrantedAccessAdapter(_)
-  ): HttpMiddleware[IO] =
+    )(implicit cs: ContextShift[IO], sync: Sync[IO]): HttpMiddleware[IO] =
```

Comment:
We always have `Sync[IO]`, no need to pass it explicitly. Same in the other places.

Reply:
Maybe but compiler says:

```
[bloop] [E] diverging implicit expansion for type cats.effect.Sync[F]                                                                                          
starting with method catsReaderWriteStateTSync in object Sync 
```

Reply:
Ok, `Sync[IO]` removed

Reply:
> Maybe but compiler says

Yeah, it tends to stumble on `blocker.delay` since it can't figure out `F` from the arguments, and the context here does not help.